docs/Developer Reference/contributing-to-chainlink.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Contributing to Chainlink"
+permalink: "docs/contributing-to-chainlink/"
+---
+Chainlink is an open-source project licensed [under the MIT license](https://github.com/smartcontractkit/chainlink/blob/master/LICENSE), and we encourage contributions from all developers and community members.
+
+[insert YT video here]
+
+# What It Means to Contribute
+When you contribute to the Chainlink project, you as a developer or community member contribute your time and effort to help improve and grow Chainlink. Your contribution can be from various methods:
+- Building and maintaining the Chainlink software and tools
+- Improving and maintaining the documentation, including translations into other languages
+- Creating Chainlink focused content (blog posts, tutorials, videos etc)
+- Becoming a developer expert
+- Running an in-person meetup or watch party
+- Running a Chainlink focused developer Bootcamp (in person or online)
+- Participate in a hackathon
+- Applying for a grant
```

Comment:
```suggestion
-[ Building and maintaining the Chainlink software and tools](https://docs.chain.link/docs/contributing-to-chainlink/#contributing-to-software-and-tooling)
- [Improving and maintaining the documentation, including translations into other languages](https://docs.chain.link/docs/contributing-to-chainlink/#contributing-to-the-documentation)
- [Creating Chainlink focused content (blog posts, tutorials, videos etc)](https://docs.chain.link/docs/contributing-to-chainlink/#creating-community-content)
- [Becoming a developer expert](https://docs.chain.link/docs/contributing-to-chainlink/#becoming-a-developer-expert)
- [Becoming a community advocate](https://docs.chain.link/docs/contributing-to-chainlink/#joining-the-chainlink-community-advocate-program)
- [Running a Chainlink focused developer Bootcamp (in person or online)](https://docs.chain.link/docs/contributing-to-chainlink/#running-a-chainlink-focused-developer-bootcamp)
- [Running an in-person meetup or watch party](https://docs.chain.link/docs/contributing-to-chainlink/#running-an-in-person-meetup-or-watch-party)
- [Participate in a hackathon](https://docs.chain.link/docs/contributing-to-chainlink/#participate-in-a-hackathon)
- [Applying for a grant](https://docs.chain.link/docs/contributing-to-chainlink/#applying-for-a-grant)
```

Reply:
Added links to the "What it means to contribute" section that sends the user to the appropriate anchor tags on the page, so they can go exactly where they want without searching the page. Also, re-ordered some items and added "becoming a community advocate" to match the order and sections on the page.

Reply:
thanks @ZakAyesh . I added your suggestions, but changed the URLs to not be absolute, ie just the anchors, so the links also work while browsing the preview site